packages/storage/src/implementation/error.ts

```diff
@@ -15,6 +14,9 @@ import { FirebaseError } from '@firebase/util';
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+import { FirebaseError } from '@firebase/util';
```

Comment:
Did the placement of this affect the build?

Reply:
No, I just cleaned it up.